src/has.ts

```diff
@@ -0,0 +1,6 @@
+import global from 'dojo-core/global';
+import { add } from 'dojo-core/has';
+
+add('history', 'history' in global);
```

Comment:
The most consistent example of prefixing we have so far is `dom-` in core and dom.  It's hard for me to figure a clear and concise hard and fast rule for this though, since we also have several unprefixed things in core, e.g. xhr/xhr2, formdata, postmessage, raf...  Not sure if any of those would also warrant prefixes.